readthedocs.yaml

```diff
@@ -5,7 +5,7 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "mambaforge-4.10"
+    python: "mambaforge-23.11"
```

Comment:
Thanks for the update @ljwoods2, 4.10 is quite old! I worry that old versions could continue to be problematic in the future though, until we reactively update the version in the pins, and doesn't help existing MDAKits. How about switching to `mambaforge-latest` so it stays up-to-date?